cluster-operator/Makefile

```diff
@@ -1,11 +1,14 @@
 PROJECT_NAME=cluster-operator
 
 docker_build: java_build
```

Comment:
Should we just remove the Docker targets here? Will they still make sense?

Reply:
If these are missing then a top-level `make all` fails:

```
make[1]: *** No rule to make target 'docker_push', needed by 'all'.  Stop.
make[1]: Leaving directory '/home/tom/messaging/strimzi/cluster-operator'
make: *** [Makefile:149: cluster-operator] Error 2
```